d/archttp/dub.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "server",
   "dependencies": {
-    "archttp": "~>1.1.1"
+    "archttp": "~>1.1.1",
+    "mir-cpuid": "~>1.2.10"
```

Comment:
is it a common usage ? I mean physical resources handling should not be a part of the framework / language ?

Reply:
Probably it will work without it. I've just took it from other repo: https://github.com/tchaloupka/httpbench/tree/master/dlang/archttp

Reply:
seems like a hack for me, I prefer no to, but let's check with  @zoujiaqing

Reply:
I propose to try with this one. In case it doesn't give any value - we can remove it.